minvws/nl-kat-coordination
# Enable/disable scheduled reports

### Changes
This PR makes it possible to disable and enable scheduled reports.
It also makes some changes in the scheduled reports table conform the new design (see ticket).

Besides this, I've created enable/disable tags that haven't been implemented as a component yet. That's why I've implemented it the same way as how it's done for the grey label ("tags-color-grey-2").

### Issue link
Closes #3124 

### Demo
https://github.com/user-attachments/assets/c06251e7-a910-4aec-a5e5-ba62477555e2

### QA notes
- Create a report and schedule it
- You'll be directed to the scheduled reports page
- Expand the row of the created report
- Click on "Disable schedule" --> Check if it works as intended
- Expand the row of the created report
- Click on "Enable schedule" --> Check if it works as intended

Check the video for an example.

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
